scripts/gen_FCHL19_rep.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python3
+import argparse
+import os
+import sys
+
+import numpy as np
+from ase.io import read, write
+from asaplib.io import str2bool
+from qml.representations import generate_fchl_acsf
+
+def repr_wrapper(frame, elements, is_periodic = False,
+                                        nRs2=24, nRs3=20,
```

Comment:
A description of these parameters would be good. If the user is allowed to change them we should include a pointer to the parameter descriptions in the qml library. If the user isn't expected to change these values we probably shouldn't be giving them the option to do so.